qemu/tests/virtio_port_hotplug.py

```diff
@@ -2,9 +2,26 @@
 import logging
 
 from avocado.utils import process
+from virttest import utils_test
+from virttest import utils_misc
 from virttest import error_context
+from qemu.tests import driver_in_use
 from qemu.tests.virtio_serial_file_transfer import transfer_data
-from qemu.tests.vioser_in_use import run_bg_test
+
+
+def continue_transfer(vm, params, sender='both'):
+    """
+    Continue transferring data via virtserialport
+
+    :param vm: VM Object
+    :param params: Dictionary with the test parameters
+    :param sender: Who send the data file
+    :return:
+    """
+    continue_time = params.get_numeric("continue_transfer_time", 10)
+    test_time = time.time() + continue_time
+    while time.time() < test_time:
+        transfer_data(params, vm, sender)
```

Comment:
Do you want to transfer data for a long time ?   For this purpose, i think enlarge the filesize is ok. The default file size is 10 now.

Reply:
Linux guest transfer data very fast so that the `bg_test_run` can't find the python process inside the guest sometimes, I will check it again whether we can set a larger file_size when testing Linux guest.